force-app/main/interoperability/classes/ProductRegistryMapper_TEST.cls

```diff
@@ -0,0 +1,90 @@
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+ * @author Salesforce.org
+ * @date 2021
+ * @group
+ * @group-content
+ * @description Test class for ProductRegistryMapper
+ */
+@isTest
+private class ProductRegistryMapper_TEST {
+    /**************************************************************************************************************************
+     * @description Test method to verify the getProductRegistrySettingsProductInformation method returns the list
+     * of ProductRegistryModel correctly
+     ***************************************************************************************************************************/
+    @isTest
+    private static void testGetProductRegistrySettingsProductInformation() {
+        String action = 'testAction';
+        String namespace = 'testNamespace';
+        String classname = 'testClassname';
+        Decimal apiVersion = 50.0;
+
+        ProductRegistryMapper.instance = new ProductRegistryMapperStub();
+        List<ProductRegistryModel> prodRegistryModelList;
+
+        Test.startTest();
+        prodRegistryModelList = ProductRegistryMapper.getInstance().getProductRegistrySettingsProductInformation();
+        Test.stopTest();
+
+        System.assert(prodRegistryModelList != null, 'prodRegistryModelList should not be null');
+        System.assert(!prodRegistryModelList.isEmpty(), 'prodRegistryModelList should not be empty');
+        System.assertEquals(1, prodRegistryModelList.size(), 'prodRegistryModelList should have 1 element');
+
+        ProductRegistryModel model = prodRegistryModelList.get(0);
+
+        System.assertEquals(action, model.action, 'action should be set with the value: ' + action);
+        System.assertEquals(namespace, model.namespace, 'namespace should be set with the value: ' + namespace);
+        System.assertEquals(classname, model.classname, 'classname should be set with the value: ' + classname);
+        System.assertEquals(apiVersion, model.apiVersion, 'apiVersion should be set with the value: ' + apiVersion);
+    }
+
+    /**************************************************************************************************************************
+     * @description Test method to comply test coverage because of with the lines not touched because of our mocked classes
+     ***************************************************************************************************************************/
+    @isTest
+    private static void testJustForCoverage() {
```

Comment:
This test is missing an assert and is not a definitive test.

Reply:
Done. Test removed. Thanks